README.md

```diff
@@ -1,42 +1,79 @@
+
 `tap-file` is a Singer tap for File.
 
 Built with the [Meltano Tap SDK](https://sdk.meltano.com) for Singer Taps.
 
-<!--
-
-Developer TODO: Update the below as needed to correctly describe the install procedure. For instance, if you do not have a PyPi repo, or if you want users to directly install from your git repo, you can modify this step as appropriate.
-
 ## Installation
 
-Install from PyPi:
-
-```bash
-pipx install tap-file
-```
-
-Install from GitHub:
+An example GitHub installation command:
 
 ```bash
-pipx install git+https://github.com/ORG_NAME/tap-file.git@main
+pipx install git+https://github.com/MeltanoLabs/tap-file.git
 ```
 
--->
-
 ## Configuration
 
 ### Accepted Config Options
 
-<!--
-Developer TODO: Provide a list of config options accepted by the tap.
-
-This section can be created by copy-pasting the CLI output from:
-
+| Setting                | Required | Default | Description |
+|:-----------------------|:--------:|:-------:|:------------|
+| protocol               | True     | None    | The protocol to use to retrieve data. One of `file` or `s3`. |
+| filepath               | True     | None    | The path to obtain files from. Example: `/foo/bar`. |
+| file_regex             | False    | None    | A regex pattern to only include certain files. Example: `.*\.csv`. |
+| cache_filepath         | False    | None    | The location to store cached files when `protocol!=file`. If left blank, caching will not be used and the entire contents of each resource will be fetched for each read operation. |
+| s3_anonymous_connection| False    |       0 | Whether to use an anonymous S3 connection, without the use of any credentials. Ignored if `protocol!=s3`. |
+| AWS_ACCESS_KEY_ID      | False    | $AWS_ACCESS_KEY_ID | The access key to use when authenticating to S3. Ignored if `protocol!=s3` or `s3_anonymous_connection=True`. Defaults to the value of the environment variable of the same name. |
+| AWS_SECRET_ACCESS_KEY  | False    | $AWS_SECRET_ACCESS_KEY | The access key secret to use when authenticating to S3. Ignored if `protocol!=s3`or `s3_anonymous_connection=True`. Defaults to the value of the environment variable of the same name. |
+| stream_maps            | False    | None    | Config object for stream maps capability. For more information check out [Stream Maps](https://sdk.meltano.com/en/latest/stream_maps.html). |
+| stream_map_config      | False    | None    | User-defined config values to be used within map expressions. |
+| flattening_enabled     | False    | None    | 'True' to enable schema flattening and automatically expand nested properties. |
+| flattening_max_depth   | False    | None    | The max depth to flatten schemas. | <!-- Manually added entries begin below. -->
+| batch_config           | False    | None    | Object containing batch configuration information. Has two child objects: `encoding` and `storage`. |
```

Comment:
Can you link to the SDK docs here for this, it would probably be helpful 🤷

Reply:
I've added a link to the Meltano documentation.